apps/01PigLatin.js

```diff
@@ -6,8 +6,102 @@ prompt.start();
 
 
 function pigLatin(word) {
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  //
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer:You want to be able to still run the app if the user types in upper case letters. This allows the user to type in upper case letter and still run the app by chainging all the inputs to lower case.
+  //
 
-    // Your code here
+
+  word  = word.toLowerCase();
+  var vowel = [a, e, i, o, u, y];
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer:
+  //
+  var vowelIndex = -1; // Set it to assume there are no vowels
+
+if (vowelIndex = -1) {
```

Comment:
**=** assigns a value to a variable. Use **===** instead. 

Good job adding an extra data validation here.